content/Foundation/Summit/2023/home.pml

```diff
@@ -34,34 +35,40 @@ Minneapolis, USA
     </h3>
   </div>
   <h3 class="highlight">
-    Register for the summit
+    Keynotes announced
   </h3>
   <p>
-    The seL4 summit will cover the complete seL4 ecosystem, consisting of the verified microkernel, as well as all seL4-related technology, tools, infrastructure, products, projects, and people.
+    We are pleased to announce two keynotes for the seL4 summit 2023.
```

Comment:
I suggest:

We are pleased to announce that the two keynotes for the seL4 summit 2023 will be Gage from NCSC and Sam Leffler from Google!

~

Then add Gage's bio, followed by:

Gage will talk about "Scoping assurance activities with seL4". 

~
Then add Gage's abstract.

Then same for Sam: bio, talk title, abstract.